index.html

```diff
@@ -21,218 +21,121 @@
-      <h2 class="card-text_title">Tonic</h2>
-      <div class="cards frame2">
-        <img src="images/frames/Frame 2.png" alt="frame2-img" />
+      <div class="card_works">
+          <div class="card_left">
+              <img src="images/snapshot/Snapshoot Portfolio(2).png" alt="snap-card">
+          </div>
+          <div class="card_right">
+              <h2>Tonic</h2>
+              <img src="images/frames/Frame 2.png" alt="">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              <div class="language_row">
+                  <ul>
+                      <li class="html"><a href="#">HTML</a></li>
+                      <li class="css"><a href="#">CSS</a></li>
+                      <li class="javascript"><a href="#">Javascript</a></li>
+                  </ul>
+                  <button class="see_button"><a href="#">See project</a></button>
+              </div>
+          </div>
       </div>
-      <p class="description-text">
-        A daily selection of privately personalized reads; no accounts or
-        sign-ups required.
-      </p>
-      <ul class="languages_list">
-        <li class="html">HTML</li>
-        <li class="css">CSS</li>
-        <li class="javascript">Javascript</li>
-      </ul>
-      <button type="button" class="see_button">See project</button>
-    </div>
 
-    <div class="card_works">
-      <img src="images/snapshot/Snapshoot Portfolio(3).png" alt="Snapshoot Portfolio" class="portfolio_img" />
-      <h2 class="card-text_title">Multi-Post Stories</h2>
-      <div class="cards frame2">
-        <img src="images/frames/Frame 2.png" alt="frame2-img" />
+      <div class="card_works">
+          <div class="card_left">
+              <img src="images/snapshot/Snapshoot Portfolio(3).png" alt="snap-card">
+          </div>
+          <div class="card_right">
+              <h2>Tonic</h2>
+              <img src="images/frames/Frame 2.png" alt="">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              <div class="language_row">
+                  <ul>
+                      <li class="html"><a href="#">HTML</a></li>
+                      <li class="css"><a href="#">CSS</a></li>
+                      <li class="javascript"><a href="#">Javascript</a></li>
+                  </ul>
+                  <button class="see_button"><a href="#">See project</a></button>
+              </div>
+          </div>
       </div>
-      <p class="description-text">
-        A daily selection of privately personalized reads; no accounts or
-        sign-ups required.
-      </p>
-      <ul class="languages_list">
-        <li class="html">HTML</li>
-        <li class="css">CSS</li>
-        <li class="javascript">Javascript</li>
-      </ul>
-      <button type="button" class="see_button">See project</button>
-    </div>
   </section>
```

Comment:
- [ ] Sorry for mentioning this late here, but please make sure to remove any code that's not related to the current milestone as this will make it difficult for the next reviewers to highlight the code related to the specific milestone when you've added all the changes at one PR. ✔